### PR TITLE
Update lzma-static.cabal

### PR DESCRIPTION
**Pull request by arjunkathuria**
needs a bytestring version bump to > 0.11.1 to make ghcup build with 9.2.1

### CODE REVIEW

File: lzma-static.cabal [hunk 1]
```diff
@@ -65,7 +65,7 @@ library
   other-modules:       LibLzma
 
   build-depends:       base       >=4.5    && <5
-                     , bytestring >=0.9.2  && <0.11
+                     , bytestring >=0.11.1
```
Comment (hasufell) [inline lzma-static.cabal]: Why `>=`?

Comment (arjunkathuria) [reply lzma-static.cabal]: base 4.16 for ghc 9.2.1

Comment (arjunkathuria) [reply lzma-static.cabal]: https://hackage.haskell.org/package/bytestring-0.11.1.0

as you can see, this is the first version that allows base > 4.15

Comment (hasufell) [reply lzma-static.cabal]: No, you're disallowing builds with older bytestrings. Why?

Comment (arjunkathuria) [reply lzma-static.cabal]: 0.11.0 is still a lower base bound.

Comment (hasufell) [reply lzma-static.cabal]: > 0.11.0 is still a lower base bound.

I don't understand. We only use `>= 0.11.1` if this packages fails to build with `< 0.11.1`. But that's not the case.

Comment (arjunkathuria) [reply lzma-static.cabal]: i see, we still want those.

so `< 0.12` should be the right thing here i guess

**Comment by hasufell** [comment]
https://hackage.haskell.org/package/lzma-static-5.2.5.4/revisions/

**Pull request merged**